src/patternfly/components/DataList/data-list.scss

```diff
@@ -80,6 +80,7 @@
   --pf-c-data-list__item-action--not-last-child--MarginRight: var(--pf-global--spacer--md);
   --pf-c-data-list__item-action--not-last-child--lg--MarginBottom: var(--pf-global--spacer--md);
   --pf-c-data-list__action--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-data-list__item-action--hidden-visible--Display: flex;
```

Comment:
how about we change this to `--pf-c-data-list__item-action--Display: flex;`, then use `var(--pf-c-data-list__item-action--Display)` for the `@include` on L182 and for the `display` property on L184?